ldi-orchestrator/ldio-common/src/main/java/be/vlaanderen/informatievlaanderen/ldes/ldio/config/LdioMetricValues.java

```diff
@@ -0,0 +1,8 @@
+package be.vlaanderen.informatievlaanderen.ldes.ldio.config;
+
+public class LdioMetricValues {
+	public static final String PIPELINE = "pipeline";
```

Comment:
In LdiSender you use PIPELINE_NAME and in LdioInput you use this. Can we be consistent with this?